source/symbols/ASTSerializer.cpp

```diff
@@ -91,6 +109,16 @@ void ASTSerializer::visit(const T& elem) {
             elem.serializeTo(*this);
         }
 
+        writer.endObject();
+    } else if constexpr(std::is_base_of_v<Statement, T>) {
+        (void)(elem);
```

Comment:
You shouldn't need this void cast, you reference the element below.

Reply:
Sorry that was used to allow compilation without implementing the `serializedTo`. I forgot to remove it.